ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -74,6 +74,8 @@ type structure_rule = {
     "not" ? boolean
     @{routing table action}
     "table" ? network_valid_routing_table
+    @{priroty also known as metric, needed by nm when setting routing rules)}
```

Comment:
```suggestion
    @{priority, also known as metric. Represents number of hops to the destination. Required by Network Manager when setting routing rules.)}
```
I assume this not required by other back-ends, so there is no need to make it required in the schema?

Reply:
correct, only used in nmstate.  default is set to 100 in the code, I didnt want to set required on schema because changing existing profiles to network backend, what do you think?

also, sorry comment wasn't correct, I fixit now, its not metric, its actually the priority of the rule over the others, so you get something like this in rule. when setting any policy rule/rourte, if that makes sense.

# ip rule list
0:      from all lookup local
100:    from 1.2.1.0/24 lookup 3 proto static
200:    from all to 1.2.3.0/24 lookup 3 proto static
32766:  from all lookup main
32767:  from all lookup default